hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/ratis/RatisHelper.java

```diff
@@ -488,4 +490,64 @@ public static void debug(ByteBuf buf, String name, Logger log) {
     }
     log.debug("{}: {}\n  {}", name, buf, builder);
   }
+
+
+  /**
+   * Use raft client to send admin request, transfer the leadership.
+   * 1. Set priority and send setConfiguration request
```

Comment:
I would suggest don't use priority to initiate transfer leadership.
Because in the next release of Ratis, we should be able to transfer leadership between nodes with same priority, see RATIS-1762 (however transfer to lower priority peer is still not allowed). So this interface won't disrupt the priority configurations.
Otherwise this interface will be deprecated soon, just like RATIS-1769.

Reply:
Hi @kaijchen. This whole process is done on the server side by ozone. Once a new version of API is coming, we can simply delete the Setting priority part and do not need to worry about the ozone client's compatibility, since the ozone client has its own PB sending the "target NodeId".

Reply:
Thanks for explaining. Since it's all happening on server side, it should be fine.